### DISCUSSION TITLE
How can I build many *.node in one platform?

### DISCUSSION THREAD

**Discussion by ouliuquan** [napi-rs/napi-rs#534] [Q&A]
As the "binding.gyp way" there are element in "target" array, one element is a *.node file, also call bindings,
Now I can't find the napi-rs way thought the online document https://napi.rs/ and ReadMe.md here.
So I have the question here, glad that you review my question.

**Comment by Brooooooklyn** [comment]
This is not possible in `Rust` world for now, because you can only define one library per project, see: https://stackoverflow.com/questions/54843118/why-can-a-cargo-package-only-have-one-library-target
